### ISSUE TITLE
Allow different layouts per inventory category

### ISSUE THREAD

**Issue by Odie** [Odie/skyui-vr#22]
As requested here:
https://www.reddit.com/r/skyrimvr/comments/9tdms2/sky_ui/e9f18zt

This probably isn't very hard to implement, but it might not be very intuitive for the user. We can make it so that every category has its own configurable layout, where columns can be enabled/disabled through a dropdown menu. But will the user understand every category is to be configured separately?

Suppose the user is on the "All" tab. He adds the val/weight column, then move to another category, say "Weapon". If each category is configured separately, the val/weight column would promptly disappear. Will this make the UI look buggy/broken?

**Comment by Odie** [comment]
Okay. What if it were easier to select/switch between columns? Say, if you were actually able to give focus to the columns and swipe through them as you would the categories? Or if there are buttons dedicated to moving the column selection left or right? Or if you can just point at the right column and have it selected?

If any of those features existed, would you still want the sortable columns to be configurable?

**Comment by baggyg** [comment]
Yes that would probably work as well if not better. The underlying problem is that it takes a a good few clicks to get things sorted by those columns. If you could easily swipe to select, then sort that should work well

**Comment by baggyg** [comment]
... laser pointing with the Vive wand would be the ultimate solution :-)

**Comment by Odie** [comment]
Yeah, that'd be really nice. Being able to physically interact with the menu might be pretty cool also. =)

**Comment by Braintelligence** [comment]
> ... laser pointing with the Vive wand would be the ultimate solution :-)
I don't know how it happened, but I once had the SkyUI menu open and also had laser pointing on a wand. I couldn't "click" on anything, tho.

I hate using the trackpad for the menu T_T...
